.github/scripts/install.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # shellcheck source=retry.sh
-source "${CI_BUILD_DIR}/.github/scripts/retry.sh"
+source "${GITHUB_WORKSPACE}/.github/scripts/retry.sh"
```

Comment:
P.S. I was trying to make everything generic enough that switching from gh actions to a different CI in the future, you'd only need to change the CI config

Reply:
_which is why everything was_

```diff
- travis
+ CI
```

Reply:
Aye, that's fine - chances of moving again are incredibly slim so I'm not too worried. As long as GA stays free and working I see no reason to switch, and by moving ci-only stuff to .github/scripts it'll help clean things up overall.

A lot of the ugliness (eg `if [[ $RUN_MODE == "npm" ]]` and other no-ops) was largely due to Travis limitations. That or limitations in my knowledge of it. Essentially it proved difficult to separate npm and php steps entirely - leading to mess everywhere - which isn't an issue in GA it seems.